tests/functional/spec/features/grand_calculated_summary/grand_calculated_summary_overlapping_answers.spec.js

```diff
@@ -40,7 +40,7 @@ describe("Feature: Grand Calculated Summary", () => {
       await click(GrandCalculatedSummaryShoppingPage.submit());
     });
 
-    it("Given I edit an answer that is only used in a single calculated summary, I am routed back to the calculated summary and then the grand calculated summary", async () => {
+    it("Given I edit an answer that is only used in a single calculated summary, I am routed back to the calculated summary and then the grand calculated summary and the correct fields are focused", async () => {
```

Comment:
I know this was pre-existing but a lot of these tests should follow the Given, When, Then pattern, a lot seem to be Given, Given 😅

Reply:
Yeah, I am not sure here, I touch only one, two assertion in each file and only one test description in all of spec files, does it mean I should change all of the test descriptions in a given spec? Keep it the same and create a separate task? Change the description to use "given, when, then" since it's one test I am editing and it will stand out?